docs/en/latest/building-apisix.md

```diff
@@ -73,7 +67,7 @@ make deps
 make install
```

Comment:
make deps is required before running make install isn't it?

Reply:
yes, but make deps is before make install for this code now, i dont get that what do u mean

Reply:
oops my bad